authorizerd.go

```diff
@@ -351,11 +351,7 @@ func (a *authorizer) verify(ctx context.Context, m mode, tok, act, res string, c
 	}
 
 	// check if exists in verification success cache
-	_, ok := a.cache.Get(tok + act + res)
-	if ok {
-		glg.Debugf("use cached result. tok: %s, act: %s, res: %s", tok, act, res)
-		return nil
-	}
+	_, isCacheHit := a.cache.Get(tok + act + res)
```

Comment:
```suggestion
	_, isCacheHit := a.cache.Get(tok + string(cert.Raw) + act + res)
```

suggestion: update key structure, keep current logic

Reply:
ok, I' ll consider using thumbprint.

Reply:
f301fa1